src/Indexer.php

```diff
@@ -60,7 +73,7 @@ public function clean(array $searchHandlers, ?bool $dryRun = null): IndexCleanRe
 
         /** @var \LoyaltyCorp\Search\Interfaces\SearchHandlerInterface[] $searchHandlers */
         foreach ($searchHandlers as $searchHandler) {
-            $handlerIndices[] = $searchHandler->getIndexName();
+            $handlerIndices = $this->indexTransformer->transformIndexNames($searchHandler);
```

Comment:
You'll need to flatten this array?

Reply:
Not really, `$handlerIndices` is still an array as default transformer returns array.

Default:
`['customers']`

Provider Specific:
`['customer_prov1', 'customer_prov2']`